backoffice/workflows/api/views.py

```diff
@@ -70,6 +80,41 @@ def create(self, request, *args, **kwargs):
             return Response({"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+class AuthorWorkflowViewSet(viewsets.ViewSet):
+    def create(self, request):
+
+        workflow = Workflow.objects.create(data=request.data["data"], workflow_type=request.data["workflow_type"])
+        serializer = WorkflowSerializer(workflow)
+
+        if workflow.workflow_type == WORKFLOW_TYPES[2][0]:
+            dag_name = "author_create_initialization_dag"
+        elif workflow.workflow_type == WORKFLOW_TYPES[3][0]:
+            dag_name = "author_update_dag"
+        else:
+            raise ValidationError("The specified workflow type is not recognized")
+
+        return airflow_utils.trigger_airflow_dag(dag_name, str(workflow.id), serializer.data)
+
+    @action(detail=True, methods=["post"])
+    def resolve(self, request, pk=None):
+
+        data = request.data
+        create_ticket = data["create_ticket"]
+        resolution = data["value"]
+        extra_data = {"create_ticket": create_ticket, "resolution": resolution}
```

Comment:
these might be error prone, why not using serializers? This is na example with a serializer https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing

We can make one serializer for author creation and one for resolving